config.mk

```diff
@@ -2,7 +2,8 @@ CABAL := cabal
 STACK := stack
 
 # Cabal Flags
-CABALFLAGS += --enable-tests --force-reinstalls
+INSTALL_FLAGS += --enable-tests --force-reinstalls
+CONFIG_FLAGS +=
```

Comment:
not sure `--enable-tests` is also needed for `CONFIG_FLAG`. certainly doesn't hurt to have.